src/Map.mo

```diff
@@ -1172,7 +1172,7 @@ module {
   ///   Map.add(map, Nat.compare, 2, "Two");
   ///
   ///   let text = Map.toText<Nat, Text>(map, Nat.toText, func (value) { value });
-  ///   // `(0, Zero), (1, One), (2, Two)`
+  ///   // `{(0, Zero), (1, One), (2, Two)}`
```

Comment:
@crusso @lucblaeser, maybe we could use a more syntax-like text representation (similar to JS and Python):

```suggestion
  ///   // `{ 0 = Zero; 1 = One; 2 = Two }`
```

Reply:
That would be easier to read, but harder to cut-n-paste into a program, i.e. as the interior of an array. Open to suggestions.
I essentially used the relation representation of map as a set of pairs.